python/GangaLHCb/test/GPI/TestExternalGaudiExec.py

```diff
@@ -110,6 +110,10 @@ def testSubmitJobComplete(self):
         from Ganga.GPI import jobs
 
         j = jobs[-1]
+        j = j.copy()
+        j.unprepare()
+        j.prepare()
+        j.submit()
```

Comment:
I wonder what an `if not j.status in ['completed', 'failed']:` check would do to help in this case as I've seen the `wait_until` scripts sometimes struggle in odd ways with already completed jobs.